### PR TITLE
Updating AMP mode dialog options and moving them to the component exa…

### PR DESCRIPTION
**Pull request by isaacgibbs**
…mples page.

| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            |  
| Patch: Bug Fix?          |
| Minor: New Feature?      |
| Major: Breaking Change?  |
| Tests Added + Pass?      | Yes
| Documentation Provided   | Yes (code comments and or markdown)
| Any Dependency Changes?  |
| License                  | Apache License, Version 2.0

Moving the AMP page property tab from the core page component definition to the example page component.

### CODE REVIEW

**Review by gaetanmarmasse — COMMENTED** [review]
@isaacgibbs , I believe this should still stay in the page properties delivered by core components. this is not implementation-specific. 
Implementations will choose to use it or not IMO. I think this PR can be canceled. 
Thoughts?

**Comment by isaacgibbs** [comment]
@gaetanmarmasse Did I misunderstand the ask in AMP-36? It seemed to indicate we didn't want this in the core components but rather in the example module. If that's not the case there's still the update to the inherit value I would like to keep and will update accordingly.

**Comment by gaetanmarmasse** [comment]
@isaacgibbs , Right, it is a little confusing. The ask is to figure out a way to tot have the option in an AMP tab in the page properties but to have it part of the "Advanced" tab for instance.

**Comment by isaacgibbs** [comment]
@gaetanmarmasse I updated the AMP selection to be part of an "Advanced" tab overlay.

File: content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/_cq_dialog/.content.xml [hunk 1]
```diff
@@ -35,32 +35,53 @@
...
                                                 sling:resourceType="granite/ui/components/coral/foundation/form/select"
                                                 composite="{Boolean}false"
-                                                fieldDescription="The mode in which you want AMP to be handled instead of what's defined in the page's content policy. Inherit from Page Template, default, will use the mode specified in the current template's page policy. Paired AMP will serve AMP pages when the 'amp' selector is present. No AMP will never serve AMP pages. AMP only will always serve AMP pages."
+                                                fieldDescription="The mode in which you want AMP to be handled instead of what's defined in the page's content policy. Paired AMP will serve AMP pages when the 'amp' selector is present. No AMP will never serve AMP pages. AMP only will always serve AMP pages."
                                                 fieldLabel="AMP Mode"
                                                 name="./ampMode">
                                                 <items jcr:primaryType="nt:unstructured">
```
Comment (gaetanmarmasse) [inline content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/_cq_dialog/.content.xml:81]: @isaacgibbs , is there a reason why we are not seeing the other modes here?
Users should be able to select "Inherit from template", "No AMP", "Paired AMP" or "AMP Only"

**Review by gaetanmarmasse — CHANGES_REQUESTED** [review]

File: content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/_cq_dialog/.content.xml (hunk 1 above)
Comment (isaacgibbs) [reply content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/_cq_dialog/.content.xml:81]: @gaetanmarmasse I'm not sure what you mean? All of those options are defined in this field.

Comment (gaetanmarmasse) [reply content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/_cq_dialog/.content.xml:81]: Yep, sorry, I was seeing a weird diff.
I've changed minor things but it looked good! :)

**Review by gaetanmarmasse — APPROVED** [review]

**Pull request merged**